README.md

```diff
@@ -3,7 +3,31 @@
 More information on this service can be found on Confluence:
 
 * https://confluence.ons.gov.uk/display/SDC/SDS
+---
+## Dockerized
+This will launch the SDS application and two emulators, the SDS application will support hot reloading within the `/src/app` directory. 
```

Comment:
What is "This"? I think there are 5 containers, not 3? The meaning is not clear.